hack/test_e2e.sh

```diff
@@ -621,7 +623,7 @@ function run_tests {
    export KUBE_CONTAINER_RUNTIME_ENDPOINT=unix:///run/containerd/containerd.sock
    export KUBE_CONTAINER_RUNTIME_NAME=containerd
 
-   ${e2e_dir}/bin/ginkgo --nodes="${GINKGO_NUMBER_OF_NODES}" \
```

Comment:
@qabpea why not just make this `"${bin_dir}/ginkgo"` instead of adding new args to run_tests and `create_infrastructure_and_run_tests`?  

Even better what if we just passed `${bin_dir}` to `run_tests()` and then that function can handle calling `gingko` and `e2e.test` from there

We keep adding arguments and things are getting confusing IMO